cmd/plume/prerelease.go

```diff
@@ -288,6 +288,12 @@ func getFedoraImageFile(client *http.Client, src *storage.Bucket, fileName strin
 	if err := sdk.UpdateFile(rawxzPath, rawxzURI.String(), client); err != nil {
 		return "", err
 	}
+
+	//decompress it
```

Comment:
Please include a space after the `//` before starting your comment.

Reply:
This was fixed in a later commit

Reply:
This still shows as no space for me.